apollo-backend/src/main/java/io/logz/apollo/blockers/types/UnconditionalBlocker.java

```diff
@@ -1,25 +1,64 @@
 package io.logz.apollo.blockers.types;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.logz.apollo.blockers.BlockerFunction;
 import io.logz.apollo.blockers.BlockerInjectableCommons;
 import io.logz.apollo.blockers.BlockerType;
-import io.logz.apollo.models.DeployableVersion;
 import io.logz.apollo.models.Deployment;
 
 import java.io.IOException;
+import java.util.List;
 
 /**
  * Created by roiravhon on 6/4/17.
  */
 @BlockerType(name = "unconditional")
 public class UnconditionalBlocker implements BlockerFunction {
+
+    private UnconditionalBlocker.UnconditionalBlockerConfiguration unconditionalBlockerConfiguration;
+
     @Override
     public void init(String jsonConfiguration) throws IOException {
-
+        ObjectMapper mapper = new ObjectMapper();
+        unconditionalBlockerConfiguration = mapper.readValue(jsonConfiguration, UnconditionalBlocker.UnconditionalBlockerConfiguration.class);
     }
 
     @Override
     public boolean shouldBlock(BlockerInjectableCommons blockerInjectableCommons, Deployment deployment) {
+
+        List<Integer> exceptionServiceIds = unconditionalBlockerConfiguration.getExceptionServiceIds();
+        List<Integer> exceptionEnvironmentIds = unconditionalBlockerConfiguration.getExceptionEnvironmentIds();
+
+        if ((exceptionServiceIds != null && exceptionServiceIds.contains(deployment.getServiceId()))
```

Comment:
What if i have excluded only service and not environment? this condition will fail